sshd.go

```diff
@@ -105,12 +105,32 @@ func (s *sshd) init(pod *Pod, config interface{}) error {
 	return nil
 }
 
-// start is the agent starting implementation for sshd.
-func (s *sshd) start(pod *Pod) error {
+// exec is the agent command execution implementation for sshd.
+func (s *sshd) exec(pod *Pod, c Container, process Process, cmd Cmd) error {
 	if pod == nil {
 		return errNeedPod
 	}
 
+	session, err := s.client.NewSession()
+	if err != nil {
+		return fmt.Errorf("Failed to create session")
```

Comment:
It would be useful to add some more detail to this error: which pod/container for example?